docs/api/members.rst

```diff
@@ -45,4 +53,5 @@ Events
 - :func:`on_member_ban(guild, user) <disnake.on_member_ban>`
 - :func:`on_member_unban(guild, user) <disnake.on_member_unban>`
 - :func:`on_presence_update(before, after) <disnake.on_presence_update>`
+- :func:`on_raw_presence_update(payload) <disnake.on_raw_presence_update>`
```

Comment:
There's another link to the non-raw event in `activities.rst`

Reply:
6740164b, thanks!